src/data/faq_de.json

```diff
@@ -467,9 +467,9 @@
-                            "Die Auslieferung der bestehenden App auf anderen Kanälen ist im Moment als Anfrage auf <a href='https://github.com/corona-warn-app/cwa-wishlist/issues/57' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-wishlist#57</a> publiziert. Die Diskussion rund um alternative Vertriebsplattformen wie F-Droid findet sich auf <a href='https://github.com/corona-warn-app/cwa-documentation/issues/5' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-documentation#5</a>.",
-                            "Weiterhin gibt es in der Entwickler-Community eine Initiative, die Corona-Warn-App über entsprechende Erweiterungen unabhängig von Google Play Services lauffähig zu bekommen. Details hierzu finden sich im Issue <a href='https://github.com/corona-warn-app/cwa-app-android/issues/75' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-app-android#75</a>."
+                            "Zum einen liegt die Verantwortung für die Auslieferung der App beim Robert Koch-Institut, welches rechtlich und inhaltlich für die Verteilung verantwortlich ist. Im Moment ist hier nur die Auslieferung über die jeweiligen App Stores im Fokus.",
+                            "Die offizielle Auslieferung der bestehenden App auf anderen Kanälen ist im Moment als Anfrage auf <a href='https://github.com/corona-warn-app/cwa-wishlist/issues/57' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-wishlist#57</a> publiziert. Die Diskussion rund um alternative Vertriebsplattformen wie F-Droid findet sich auf <a href='https://github.com/corona-warn-app/cwa-app-android/issues/1483' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-app-android#1483</a>.",
+                            "Von der Entwickler-Community wird ein inoffizieller Fork der App unter dem Namen <i>Corona Contact Tracing Germany</i> entwickelt, welcher die Corona-Warn-App über entsprechende Erweiterungen unabhängig von Google Play Services lauffähig macht. Die Entwicklung des Forks wird jedoch nicht vom Robert Koch-Institut betreut und erhält daher auch <b>keinen offiziellen Support</b>. Details zu der Entwicklung finden sich im Issue <a href='https://github.com/corona-warn-app/cwa-app-android/issues/75' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-app-android#75</a> sowie im Code Repository des Forks: <a href='https://codeberg.org/corona-contact-tracing-germany/cwa-android' target='_blank' rel='noopener noreferrer'>Corona Contact Tracing Germany</a>."

```

Comment:
Change "Google Play Services" to German name "Google Play-Dienste"

(see https://play.google.com/store/apps/details?id=com.google.android.gms&hl=de)

Reply:
fixed